src/components/common/accountPinModal/UserForm.js

```diff
@@ -24,6 +26,9 @@ function UserForm({
       pinForm: true,
     });
   };
+
+  useEffect(() => {}, [isLoading]);
```

Comment:
What is this `useEffect` supposed to be doing?

Reply:
i was using it to force a re-render, but no longer needed. Thanks for pointing this out.